app/assets/stylesheets/common.scss

```diff
@@ -546,6 +547,13 @@ body.compact {
     background: #fff;
     font-size: 12px;
 
+    > div {
+      position: relative;
+      float: left;
+      clear: both;
+      width: 100%;
+    }
+
```

Comment:
Oh sure, my meaning was more what divs do we actually create there already if any and what impact does this have on them...